refarch-eai/pom.xml

```diff
@@ -20,13 +21,11 @@
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
         <maven.compiler.release>${java.version}</maven.compiler.release>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven-scm-plugin.version>1.10.0</maven-scm-plugin.version>
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
 
         <!-- Core Frameworks -->
         <camel-spring-boot-dependencies.version>4.1.0</camel-spring-boot-dependencies.version>
```

Comment:
We should add a note stating this must match the spring-boot version provided in the parent.